pkg/api/types.go

```diff
@@ -1662,6 +1662,83 @@ type SecretList struct {
 	Items []Secret `json:"items"`
 }
 
+// SecurityContext specifies the security constraints associated with a service account
+type SecurityContext struct {
+	TypeMeta   `json:",inline"`
+	ObjectMeta `json:"metadata,omitempty"`
```

Comment:
This is not an ObjectMeta.  There are two security context structs in play - one that would be part of a Container definition (or maybe a pod) and one that would be part of a Service Account.  There is no independent security context object.